test/integration/forms/patient.js

```diff
@@ -4,12 +4,11 @@ import dayjs from 'dayjs';
 import formatDate from 'helpers/format-date';
 import { testDate, testDateSubtract } from 'helpers/test-date';
 import { testTs, testTsSubtract } from 'helpers/test-timestamp';
-import { getRelationship, getErrors } from 'helpers/json-api';
```

Comment:
I think we might need more linting errors to catch this on the PR

Reply:
I just added the free-for-open-source coderabbit.ai which will probably catch this.